tidb-server/main.go

```diff
@@ -525,6 +531,15 @@ func overrideConfig() {
 	}
 }
 
+func setSafeModeConfig() {
+	if cfg.Store != "mocktikv" {
+		cfg.RunDDL = false
+	}
```

Comment:
Is it needed to add some mem quota and token limits?

Reply:
@coocood suggests that it is no need to add limits for token. 😄